src/test/java/io/nats/client/impl/ReconnectTests.java

```diff
@@ -298,16 +266,16 @@ public void testMaxReconnects() throws Exception {
                                     reconnectWait(Duration.ofMillis(10)).
                                     build();
                 nc = Nats.connect(options);
-                assertTrue(Connection.Status.CONNECTED == nc.getStatus(), "Connected Status");
+                assertConnected(nc);
                 handler.prepForStatusChange(Events.CLOSED);
             }
 
             flushAndWait(nc, handler);
-            assertTrue(Connection.Status.CLOSED == nc.getStatus(), "Closed Status");
+            assertSame(Connection.Status.CLOSED, nc.getStatus(), "Closed Status");
```

Comment:
assertSame is recommended for enum, even though it does the exact same thing. Intellij called it "simplify"